src/test/groovy/testutil/PluginSpecification.groovy

```diff
@@ -0,0 +1,28 @@
+package testutil
+
+import org.gradle.testfixtures.ProjectBuilder
+import org.junit.Rule
+import org.junit.rules.TemporaryFolder
+import spock.lang.Specification
+
+class PluginSpecification extends Specification{
```

Comment:
I like the new class a lot! It also hides other complexities like 'tmp' dir, creation of 'project'.

Let's add some Javadocs on why and how